themes/default/content/blog/pulumi-release-notes-71/index.md

```diff
@@ -0,0 +1,148 @@
+# if you don't already have one.
+authors:
+    - meagan-cojocar
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Our first release notes since the frenzy of [releases for PulumiUP](/blog/pulumi-universal-iac)! Learn what we have been building in the past month.
+
+ <!--more-->
+- Cloud Providers and Packages
+  - [New resources in our providers](#new-resources-in-our-providers)
+- Pulumi CLI and core technologies
+  - [Pulumi YAML v0.5.1](#pulumi-yaml-v051)
+  - [Add --stack to `pulumi about`](#add---stack-to-pulumi-about)
+  - [Add logout message](#add-logout-message)
+  - [Warn about missing AdditionalSecretOutputs](#warn-about-missing-additionalsecretoutputs)
+  - [Compression of filestate backends](#compression-of-filestate-backends)
+  - [Add `CompositeInvoke`](#add-compositeinvoke)
+- Pulumi Service & Pulumi.com
+  - [Stack READMEs](#stack-readmes)
+  - [SAML/SCIM improvements](#samlscim-improvements)
+
+## Cloud Providers and Packages
+
+### New resources in our providers
+
+We shipped new versions of the AWS Native provider, Google Native provider and the Azure Native provider that added support for 412 new resources in the last month. 385 of those resources were Azure Native, 7 AWS Native and 20 were added to Google Native.
+
+## Pulumi CLI and core technologies
+
+### Pulumi YAML v0.5.1
+
+We released v0.5.1 of Pulumi YAML which included bug fixes, new functions, diagnostics and validation. Some specific improvements we made were:
+
+- Update pulumi/pulumi to v3.32.1
+
+- Add errors when hanging invalid fields off of resources.
+  [#203](https://github.com/pulumi/pulumi-yaml/pull/203)
+
+- Add errors when hanging invalid fields off of resource options.
+  [#211](https://github.com/pulumi/pulumi-yaml/pull/211)
+
+- Add a type checker.
+  [#228](https://github.com/pulumi/pulumi-yaml/pull/228)
+
+- Add `Fn::FromBase64`
+  [#218](https://github.com/pulumi/pulumi-yaml/pull/218)
+
+- Add support for Fn::ReadFile, enabling Pulumi Service [Stack README]({{< relref "/blog/stack-readme" >}}) support.
+  [#217](https://github.com/pulumi/pulumi-yaml/pull/217)
+
+- Allow Fn::Join to take expressions as inputs, previously the second argument had to be a syntactical list.
+  [#241](https://github.com/pulumi/pulumi-yaml/pull/241)
+
+As always, please feel free to submit feature requests and bug reports to the [Pulumi YAML GitHub Repo](https://github.com/pulumi/pulumi-yaml). We love hearing feedback from users!
+
+### Add --stack to `pulumi about`
+
+You can now use `pulumi about --stack` to get information on your stacks. It defaults to the current stack but you can specify the stack you want information on, for example `pulumi about --stack eks/staging`.
```

Comment:
Perhaps something like "The `pulumi about` commands lets you ....  We added the ability to specify which stack to provide details about without having to select it first using `pulumi about --stack`."